src/status_im2/contexts/wallet/account/tabs/view.cljs

```diff
@@ -1,13 +1,57 @@
 (ns status-im2.contexts.wallet.account.tabs.view
   (:require
     [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
     [react-native.core :as rn]
     [status-im2.contexts.wallet.account.style :as style]
     [status-im2.contexts.wallet.common.temp :as temp]
-    [utils.i18n :as i18n]))
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn description
+  []
+  [quo/text {:size :paragraph-2}
+   (map (fn [network]
+          ^{:key (str network)}
+          [quo/text
+           {:size   :paragraph-2
+            :weight :medium
+            :style  {:color (get colors/networks network)}}
```

Comment:
please use the custom-color method for this instead. it abstracts all of the colors system away then 👍 
see:
https://www.notion.so/Colors-mechanism-d4060bc87cfb4c268d19a77f0711d3a1?pvs=4#7f28bca036f44f79ad973997aad2b849

e.g `(colors/custom-color  network)`